src/Workload/Microsoft.Maui.Dependencies/Microsoft.Maui.Dependencies.csproj

```diff
@@ -13,6 +13,10 @@
   <Import Project="../Shared/LibraryPacks.targets" />
 
   <ItemGroup>
+    <PackageReference Include="Microsoft.Extensions.Configuration" />
+    <PackageReference Include="Microsoft.Extensions.DependencyInjection" />
+    <PackageReference Include="Microsoft.Extensions.Logging" />
+    <PackageReference Include="Microsoft.Extensions.Logging.Abstractions" />
```

Comment:
@pjcollins another idea is to not put these here and instead...

Reply:
Should the same be done for Dependencies at this point? It seems like we shouldn't need a library pack declaring any external package references when we could declare them in our own targets?